Tests/ApiEndpoints/test_gordon360_pytest.py

```diff
@@ -72,22 +85,97 @@ def createAuthorizedSession(self, userLogin, userPassword):
         authorized_session.headers.update({ "Authorization":authorization_header })
         return authorized_session
 
+
+class Test_allScheduleControlTest(testCase):
+
+# # # # # # # # # # # # #
+# SCHEDULECONTROL TESTS #
+# # # # # # # # # # # # #
+
+#    Get the schedulecontrol object of the currently logged in user.
```

Comment:
What is schedulecontrol object??
Be specific about what type of information it returns

Reply:
added